### PR TITLE
Kaggle Community Connector

### PR DESCRIPTION
**Pull request by surajsnair92**

### CODE REVIEW

File: kaggle/src/connector.js [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+  return request;
+}
+function getData(request) {
+  request = validateConfig(request);
+  var result = getFileData(request.configParams);
+  var rawData = result.csvData;
+  var cacheKey = result.cacheKey;
+  var cache = CacheService.getScriptCache();
+  var cachedSchema = JSON.parse(cache.get(cacheKey))
+  if(cachedSchema === null)
```
[the remark this answers is not in the document]
Comment (diminishedprime) [reply kaggle/src/connector.js]: Also make sure to run `prettier` on this code and commit the changes.

[the remark this answers is not in the document]
Comment (surajsnair92) [reply kaggle/src/connector.js]: done!

File: kaggle/README.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+
+## Deploy the Community Connector yourself
+
+Use the [deployment guide](../deploy.md) to deploy the Community Connector
+yourself.
+
+## Examples and use cases covered in the connector
+
+- **Kaggle authentication required**  
+  This community connector requires third-party authentication. To use the Kaggle API, sign up for a Kaggle account [kaggle](https://www.kaggle.com). Then go to the 'Account' tab of your user profile (https://www.kaggle.com/{username}/account) and select 'Create API Token'. This will trigger the download of kaggle.json, a file containing your API credentials.
```
[the remark this answers is not in the document]
Comment (surajsnair92) [reply kaggle/README.md]: done!

**Comment by diminishedprime** [comment]
Good job! Thanks for the PR.

**Pull request merged**